docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
+right of the search bar. The wizard guides you through the following steps:
+
+[float]
+[[newvis01]]
+==== Step 1: Choose the Visualization Type
+
+The New Visualization wizard starts with the following screen:
+
+image:images/NewViz01.png[]
+
+The data table, Markdown, and metric elements aren't visualizations, but all the elements on the list can be included 
+in a <<dashboard,Dashboard>>.
+
+You can also open a saved visualization that you created earlier. The saved visualization selector includes a text 
+field to filter by visualization name and a link to the Object Editor, accessible through *Settings > Edit Saved 
+Objects*, to manage your saved visualizations.
+
+If your new visualization is a Markdown widget, selecting that type takes you to a text entry field where you enter the 
+text to display in the widget. For all other types of visualization, selecting the type takes you to data source 
+selection.
+
+[float]
+[[newvis02]]
+==== Step 2: Choose a Data Source
+
+You can choose a new or saved search to serve as the data source for your visualization. Searches are based on an 
+_index pattern_, which is a regular expression that matches the indices you want to visualize. After selecting _new 
+search_, select an index pattern from the drop-down to bring up the visualization editor. 
+
+// How is this drop-down populated? Is it just a list of all indices in the cluster? Can I configure the contents?
+
+When you create a visualization from a saved search and save the visualization, the search is tied to the visualization.
+When you make changes to the search that is linked to the visualization, the visualization updates automatically.
+
+[float]
+[[visualization-editor]]
+==== Step 3: The Visualization Editor
+
+The visualization editor enables you to configure and edit visualizations. The visualization editor has the following 
+main elements:
+
+image:images/VizEditor.png[]
+
+1. <<toolbar-panel,Toolbar Panel>>
+2. <<aggregation-builder,Aggregation Builder>>
+3. <<preview-canvas,Preview Canvas>>
+
+[float]
+[[toolbar-panel]]
+===== Toolbar Panel
+
+The toolbar panel has a search field for interactive data searches, as well as toolbars to manage saving and loading
+visualizations. For visualizations based on saved searches, the search bar is initially disabled. Double-click the 
+grayed-out search field to enable interactive searching.
+
+// Why does it behave this way? I'd like to be able to say 'for saved searches interactive searches are disabled
+// because $REASONS'.
+
+The toolbar at the right of the search box has buttons for creating new visualizations, saving the current 
+visualization, loading an existing visualization, sharing or embedding the visualization, and refreshing the data for 
+the current visualization.
+
+[float]
+[[aggregation-builder]]
+===== Aggregation Builder
+
+Use the aggregation builder on the left of the screen to configure the 
```

Comment:
I'd be inclined to use "page" instead of "screen". I think of screen as my entire display, and the contents displayed within a web browser as a "page".